platzky/blog/blog.py

```
@@ -13,7 +15,8 @@
 logger = logging.getLogger(__name__)
 
 
-def create_blog_blueprint(db, blog_prefix: str, locale_func):
+def create_blog_blueprint(db: Any, blog_prefix: str, locale_func: Callable[[], str]) -> Blueprint:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `db`

(ANN401)